kinto/config-check-test/test_kinto_signer.py

```diff
@@ -35,29 +35,26 @@ def verify_signatures(client):
         return 0
 
 
+def get_signer_id(client):
+    dest_col = client.get_collection()
```

Comment:
I was not aware that in python I could pass back multiple values like that. I reworked the code along these lines.

r? @Natim